uaa/src/test/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpointsAliasMockMvcTests.java

```diff
@@ -69,6 +80,11 @@ void setUp() throws Exception {
         scimUserService = requireNonNull(webApplicationContext.getBean(ScimUserService.class));
     }
 
+    @AfterEach
+    void tearDown() {
+        jdbcTemplate.execute("DELETE FROM users WHERE username = '" + TEST_USERNAME + "'");
```

Comment:
do we need the deletion if we use random user names ?

Reply:
Very good point. It is not required. 

Since there are many tests, it's useful when debugging to have less users with random names in the table. I'm going to add a comment.